tools/build-overrides/build-overrides.sh

```diff
@@ -458,8 +450,7 @@ EOF
                     log_info "Generating ${kieserver_overrides_yaml} ..."
 cat <<EOF > "${kieserver_overrides_yaml}"
 artifacts:
-- name: "KIE_SERVER_DISTRIBUTION_ZIP"
-  target: "kie_server_distribution.zip"
+- name: "rhpam_kie_server_distribution.zip"
```

Comment:
This is RHDM Kie Server (see https://github.com/jboss-container-images/jboss-kie-modules/pull/407/files#diff-34d9276ab4d4a89c1f08e5458bb8e87ab91a38c7e611a20f8381087b1e92c073R439)
```suggestion
- name: "rhdm_kie_server_distribution.zip"
```